thread_win32.c

```diff
@@ -777,4 +777,26 @@ native_set_thread_name(rb_thread_t *th)
 {
 }
 
+static unsigned long __stdcall
+mjit_worker(void *arg)
+{
+    void (*worker_func)(void) = arg;
```

Comment:
Add following:
```c
rb_w32_set_thread_description(GetCurrentThread(), L"ruby-mjitworker");
```
[SetThreadDescription](https://msdn.microsoft.com/en-us/library/windows/desktop/mt774976(v=vs.85).aspx) doesn't have the max size limit but I think the name should equal to Unix.

Reply:
fixed at https://github.com/ruby/ruby/pull/1782/commits/ddb2ff7303982b19adb0a23707722e3ca4ece1d2